Azure.Core.All.sln

```diff
@@ -0,0 +1,3076 @@
+﻿
```

Comment:
This should not be checked in. If this is the one that gets generated (@pakrym?), we should add this specific file path to .gitignore to make sure this doesn't happen.

Reply:
Strange, I've had this file for a while on my local clone. I'll add it to the .gitignore

Reply:
It's gitignored inside core https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/.gitignore#L1 I didn't realize the script would create solution file in the current directory.

Reply:
Hmm, it's already there. Any ideas why it is showing up here?
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/.gitignore

Reply:
Your azure.core.all is in the root of the repo

Reply:
Oh, I was confused because I also have one in core/Azure.Core